chain/chain/src/state_snapshot_actor.rs

```diff
@@ -46,13 +46,15 @@ pub struct DeleteAndMaybeCreateSnapshotRequest {
 #[derive(actix::Message)]
 #[rtype(result = "()")]
 pub struct CreateSnapshotRequest {
-    /// prev_hash of the last processed block.
+    /// equal to self.block.header().prev_hash()
     prev_block_hash: CryptoHash,
+    /// Min height of chunk.prev_block_hash() for each chunk in `block`
+    min_chunk_prev_height: BlockHeight,
     /// epoch height associated with prev_block_hash
     epoch_height: EpochHeight,
     /// Shards that need to be present in the snapshot.
     shard_indexes_and_uids: Vec<(ShardIndex, ShardUId)>,
-    /// Last block of the prev epoch.
+    /// prev block of the "sync_hash" block.
     block: Block,
```

Comment:
nit: might even called it `prev_sync_hash`

Reply:
yea I thikn in another PR I will remove the unnecessary prev_block_hash field and then we can rename this one